.woodpecker.star

```diff
@@ -1199,7 +1199,8 @@ def e2eTestPipeline(ctx):
     }
 
     extra_server_environment = {
-        "OCIS_PASSWORD_POLICY_BANNED_PASSWORDS_LIST": "%s" % dirs["bannedPasswordList"],
+        "OC_PASSWORD_POLICY_BANNED_PASSWORDS_LIST": "%s" % dirs["bannedPasswordList"],
+        "OC_SHOW_USER_EMAIL_IN_RESULTS": True,
```

Comment:
the tests type the whole email address and not only the username into the sharing fields. I believe that is the case since https://github.com/opencloud-eu/web/pull/32
CC @ScharfViktor

Reply:
yeah, we should have this env to find users